### DISCUSSION TITLE
Schema for the Motion DB tables?

### DISCUSSION THREAD

**Discussion by davidd8** [filecoin-project/motion#13] [General]
Starting a thread to discuss the table schema for Motion. It seems we will need:
- CID table for local caching until blob is stored on Filecoin. Columns include: CID (dataID), Blob, PieceCID, timestamp, 
- Replicas table for each deal made for the dataID. Columns include: dataID, PieceCID, DealID, SP ID, timestamp, deal_status
- Policy table for dataID (or client). Columns include: replication factor, geo restrictions, etc.

**Comment by masih** [comment]
There is an ongoing domain modeling process kindly started by @hannahhoward. While this work is in progress, is there a particular aspect you'd like to discuss or delve into further? I imagine we can't make much progress on the concrete database schema until domain modeling reaches a critical mass.
